src/FlaUILibrary/flaui/module/screenshot.py

```diff
@@ -107,6 +125,33 @@ def _capture(self):
 
         return filepath
 
+    def _capture_base64(self, xpath=None):
+        image = None
+
+        try:
+            if xpath:
+                image = Capture.Element(self._automation.GetDesktop().FindFirstByXPath(xpath))

```

Comment:
Do not use here automation.GetDesktop usage because this will force unknown error handling behavior like COMExceptions.

Resolving from element should be executed in Keyword code block from Snapshots

Reply:
done